extension/experiments/remotesettings/api.js

```diff
@@ -11,10 +11,13 @@ const { ExtensionError } = ExtensionUtils;
 const SERVER_LOCAL = "http://localhost:8888/v1";
 const SERVER_PROD = "https://firefox.settings.services.mozilla.com/v1";
 const SERVER_STAGE = "https://settings-cdn.stage.mozaws.net/v1";
+const SERVER_DEV = "https://settings.dev.mozaws.net/v1";
 const HASH_PROD =
   "97:E8:BA:9C:F1:2F:B3:DE:53:CC:42:A4:E6:57:7E:D6:4D:F4:93:C2:47:B4:14:FE:A0:36:81:8D:38:23:56:0E";
 const HASH_STAGE =
   "3C:01:44:6A:BE:90:36:CE:A9:A0:9A:CA:A3:A5:20:AC:62:8F:20:A7:AE:32:CE:86:1C:B2:EF:B7:0F:A0:C7:45";
+// DEV signing will use STAGE's root hash
```

Comment:
It would be more correct to say that *DEV signing will use Autograph STAGE*

Reply:
Fixed!